docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: guide
+summary_title: "Codewind in CodeReady Workspaces"
+title: "Getting Started with Codewind in CodeReady Workspaces"
+categories: guides 
+guide_description: "Use Codewind in CodeReady Workspaces to build high quality cloud native applications."
+permalink: codewind-crw-quick-guide.html
+duration: 10 minutes
+tags: Codewind, CodeReady Workspaces, Openshift, Kubernetes, containers, microservice
+--- 
+
+## Objectives
+* Install CodeReady Workspaces and Codewind
+* Develop a simple microservice that uses Eclipse Codewind on CodeReady Workspaces
+
+## Overview
+Use Eclipse Codewind to develop microservice applications from application stacks in an integrated developer environment (IDE). CodeReady Workspaces provides a containerized IDE for cloud native application development on an OpenShift cluster. 
+
+## Developing with CodeReady Workspaces 
+CodeReady Workspaces uses Kubernetes and containers to provide a preconfigured environment. Use CodeReady Workspaces to create, build, and test your code in OpenShift containers but feel like you are working on an IDE on your local machine.  
+
+### Prerequisite
+CodeReady Workspaces require at least two 5Gi ReadWriteOnce (RWO) persistent volumes on the cluster to install and a 5Gi RWO volume for each created workspace.
+
+Each Codewind workspace also requires at least on 5Gi ReadWriteMany (RWX) persistent volume.
+
+Before you can use Codewind with CodeReady Workspaces, apply the Codewind cluster role with `oc apply -f`. 
+
+### Installing CodeReady Workspaces
+1\. Subscribe to the CodeReady Workspaces OperatorHub then install CodeReady Workspaces:
+
+![Subscribe to OperatorHub](./images/codereadyworkspaces-images/CRW-OperatorHub.png){:width="1684"}. 
+
+![Install CodeReady Workspaces](./images/codereadyworkspaces-images/CRW-install.png){:width="1626"}. 
+
+  * If you did not set up an auth provider, set `openShiftoAuth` to `false`. 
+  * To enable HTTPS, set `tlsSupport` to `true`.
+  * If you use a self-signed certificate, set `selfSignedCert` to `true`. 
+
+2\. Use the following yaml:  
```

Comment:
Another good point Naveen raised, for this yaml file, do we save the file and use `oc apply -f <file>`?

Reply:
To install CRW (once its operator has been installed):
1) From the OpenShift dashboard, go to `Operators` -> `Installed Operators`
2) Select `CodeReady Workspaces`
3) Select `CodeReady Workspaces` cluster. Then `Create CheCluster`
4) Copy and paste the yaml into the text box you see. And configure the fields listed above (such as `tlsSupport` as needed.

Reply:
@johnmcollier I don't see the CodeReady workspace in my console. Is there a way to install it?

Reply:
@naveenkaratekid What version of OCP are you testing against? 4.x or higher will be needed

Reply:
@johnmcollier I believe I'm running OCP 4.3 from FYRE

Reply:
@naveenkaratekid Have you tried the steps in https://github.com/eclipse/codewind-docs/pull/613#discussion_r436721067?

Reply:
@johnmcollier I just ran these two commands:
```
oc adm policy add-scc-to-user anyuid system:serviceaccounts:default:che-workspace
oc adm policy add-scc-to-user privileged system:serviceaccounts:default:che-workspace
```
The docs don't clearly explain how to start codewind using codeready. The guide says 
```
CodeReady Workspaces starts Codewind and installs the Codewind plug-ins. This process might take a couple of minutes for all of the necessary components to be pulled and started.

But after I refreshed the console, and clicked Resources, I didn't see Codewind show up there
```

Reply:
@naveenkaratekid Once you've installed CodeReady Workspaces, you need to log in and create a Codewind workspace from the Codewind devfile, as you would for Che (basically https://www.eclipse.org/codewind/che-createcodewindworkspace.html)

@j-c-berger Maybe we need to add the steps from https://www.eclipse.org/codewind/che-createcodewindworkspace.html in to this guide?

Reply:
@johnmcollier, I'm happy to include these steps. I'm thinking we should include these new steps before we say, `CodeReady Workspaces now installs and you can access the Codewind Workspace URL and Red Hat SSO Admin Console URL`. So putting these new steps after the yaml file. 

Agree?